roles/cockpit-ui/tasks/main.yml

```diff
@@ -1,63 +1,70 @@
 ---
-- block:
+- name: Create local temp dir for registry-console template
+  local_action: command mktemp -d /tmp/openshift-ansible-XXXXXXX
```

Comment:
This should be a proper `command`, as the template is being copied to master

Reply:
Good catch.